.circleci/config.yml

```
@@ -638,7 +638,7 @@ workflows:
     triggers:
       - schedule:
           # UTC time-zone
-          cron: "00 16 * * 2"
+          cron: "45 16 * * 3"
```

```suggestion
          cron: "00 16 * * 2"
```
we should change this back right?